src/deepsparse/server/main.py

```diff
@@ -152,7 +152,7 @@ def _create_endpoint(endpoint_path: str, from_files: bool = False):
         async def _predict_func(request: input_schema):
             if from_files:
                 request = pipeline.input_schema.from_files(
-                    file.file for file in request
+                    [file.file for file in request], from_server=True
```

Comment:
```suggestion
                    (file.file for file in request), from_server=True
```
Generator will be less expensive than a `list`

Reply:
classic @rahul-tuli

Reply:
:)